.github/actions/ppml-spark-local-simplequery-tests-on-gramine-action/action.yml

```diff
@@ -0,0 +1,58 @@
+name: 'PPML-spark-Local-SimpleQuery-Tests-on-Gramine'
```

Comment:
Could you put this action into .github/actions/ppml/ folder? 
`.github/actions/ppml/ppml-spark-local-simplequery-tests-on-gramine-action/action.yml`